tests/unit/database/test_identifiers.py

```diff
@@ -21,10 +21,43 @@
 
 import pytest
 
```

Comment:
Do we want to keep the tests under `tests/unit/database/test_identifiers.py` or transfer them to `tests/unit/database/test_article.py` ?

Reply:
Moved under `test_article.py` in 08e5c03.